recipes/yaml-cpp/all/conanfile.py

```diff
@@ -121,7 +121,7 @@ def _module_file_rel_path(self):
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "yaml-cpp")
-        self.cpp_info.set_property("cmake_target_name", "yaml-cpp")
+        self.cpp_info.set_property("cmake_target_name", "yaml-cpp::yaml-cpp")
```

Comment:
this is wrong

Reply:
Ah, there's no namespace in the export... I assumed they'd export the `yaml-cpp::yaml-cpp` since it was already available as an alias: https://github.com/jbeder/yaml-cpp/blob/7d066556112e6899783608ef917b9b326fb6497a/CMakeLists.txt#L83.

Does CMakeDeps need to provide this target for compatibility or should I just close this PR?

Reply:
What you could eventually do is to provide an alias target (`cmake_target_aliases` property), seems reasonable. And the best would be to submit a PR upstream, because if they want to be good citizen ALIAS target should match exported target.

Reply:
I'll go ahead and close this for now then.